Tests/Marketplace/upload_packs.py

```diff
@@ -467,10 +467,12 @@ def update_index_with_priced_packs(private_index_path: str, extract_destination_
 
         logging.info("add_private_packs_to_index")
         add_private_packs_to_index(index_folder_path, private_index_path)
-        logging.info("Finished updating index with priced packs")
-    except Exception:
-        logging.exception('Could not add private packs to the index.')
+
+    except Exception as e:
+        logging.exception(f"Could not add private packs to the index. Additional Info: {str(e)}")
+
     finally:
+        logging.info("Finished updating index with priced packs")
```

Comment:
What do you say about:
```suggestion
        logging.info("Finished creating index with priced packs")
```

Reply:
> @guyfreund maybe we should add to this PR the following task: [demisto/etc#34007](https://github.com/demisto/etc/issues/34007)
> What do you say? I can add it

Let's leave it here

Reply:
Index is not being created here